docs/docs/setup/theme.md

```diff
@@ -21,14 +21,53 @@ Once the logo file is created and configured, adjust `header_height` to your lik
 This is the height of the header (and thus also your logo) in pixels.
 Only the logo is stretched, all other elements are vertically centered within the header.
 
-You can also configure a second logo file as `logo.small` which is used for narrow screens (e.g. phones).
-This is usually roughly square.
-We strongly recommend setting this smaller logo, as otherwise, the main logo (especially if it is very wide) might get shrunk on narrow screens in order to still show the other elements in the header.
-
-You should also test if the logo is  properly configured for dark mode:
-- To use a different image for dark mode, set `logo.large_dark` and `logo.small_dark` appropriately.
-- If your normal logo already works well for dark mode, set `logo.large_dark` and `logo.small_dark` to the same values as `large` and `small`, respectively.
-- If `logo.large_dark` and `logo.small_dark` are not set, `large` and `small` are used, but with all colors inverted. This might work for you in special cases, e.g. if your logo is fully black or transparent.
+
+You can configure different logo files for different cases, depending on device-size (mobile vs desktop), color scheme (light vs dark) and language. In the simplest case of using a single logo for all cases, your config looks like this:
```

Comment:
I would suggest a newline after each sentence (compare the rest of the docs). Otherwise lines get very long. That has no effect on the rendered docs, just the source files.